tree/ntuple/v7/test/ntuple_basics.cxx

```diff
@@ -19,6 +19,14 @@ TEST(RNTuple, ReconstructModel)
 
    RPageSourceFile source("myNTuple", fileGuard.GetPath(), RNTupleReadOptions());
    source.Attach();
+   try {
+      source.SetEntryRange({0, 2});
```

Comment:
Perhaps change to
```suggestion
      source.SetEntryRange({0, source.GetNEntries() + 1});
```
to make it match with ll. 29 and make the expected error more clear.